.yamllint

```diff
@@ -6,17 +6,25 @@ ignore: |
   .github/
   # Generated file
   tests/files/custom_cni/cilium.yaml
-
+# https://ansible.readthedocs.io/projects/lint/rules/yaml/
 rules:
   braces:
     min-spaces-inside: 0
     max-spaces-inside: 1
   brackets:
     min-spaces-inside: 0
     max-spaces-inside: 1
+  comments:
+    # https://github.com/prettier/prettier/issues/6780
+    min-spaces-from-content: 1
+  # https://github.com/adrienverge/yamllint/issues/384
```

Comment:
I don't think the reference to the issue has any added value here, could you remove it?

Reply:
I meant the prettier one for `min-spaces-from-content` I commented the wrong line sorry